prog/vm/github_runner.rb

```diff
@@ -223,6 +223,33 @@ def setup_info
       UBICLOUD_CACHE_URL=#{Config.base_url}/runtime/github/" | sudo tee -a /etc/environment
     COMMAND
 
+    if vm.vm_host.id == Vm[Config.docker_mirror_server_vm_id]&.vm_host&.id
+      mirror_server_vm = Vm[Config.docker_mirror_server_vm_id]
+      mirror_server_address = mirror_server_vm.load_balancer.hostname
+      mirror_server_port = 5000
+      command += <<~COMMAND
```

Comment:
When we run multiline commands, we usually add `set -ueo pipefail` at the beginning of the commands. Otherwise, the commands will keep running even if one of the middle ones fails.

Reply:
As I'm appending to the command, `set -ueo pipefail` is already there.